packages/@lwc/style-compiler/typings/cssnano-preset-default.d.ts

```diff
@@ -1,16 +0,0 @@
-/*
```

Comment:
Update `packages/@lwc/style-compiler/tsconfig.json` to remove `typings` from `includes` folder.